### PR TITLE
Deploy scripts

### PR DESCRIPTION
**Pull request by WillMcIntosh**
Addresses issue #60 
I did this a few ways before settling on this way, which is by far the simplest. The node package gh-pages will default publish to remote/origin, which will be the user's fork if you forked the codeforboston repo as the readme instructions say to. Running `npm run deploy` will deploy to your forked repo, creating a gh-pages branch if you don't already have one. 

If you run `npm run deploy-dev` it will (theoretically) deploy to Code For Boston's gh-page just as `npm run deploy` did before. I haven't tried it because I didn't want to accidentally deploy without someone checking my code first. 

I can use a different approach if we want this to be more secure. This currently still doesn't prevent someone from just cloning the cliff-effects repo and deploying straight to it, but I think you have to be a contributor to do that if I'm not mistaken.

### CODE REVIEW

**Comment by WillMcIntosh** [comment]
This doesn't seem to be working the way I thought it was after further testing with @knod. I'll push updates to my branch after troubleshooting more.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "deploy": "npm run build&&gh-pages -d build"
+    "deploy-dev": "npm run build && gh-pages -d build",
```
Comment (knod) [inline package.json]: Tested, but not yet working. Some issues have come up with deployment that we're trying to track down.

**Comment by WillMcIntosh** [comment]
This actually seems to be a non issue... @danaher2 and I just tested it and if you fork the code for boston repo like the readme says, `npm run deploy` actually deploys to your personal gh-pages branch. You have to have a clone of the actual Code For Boston repo to deploy to the production site. Should we close this issue?

**Comment by WillMcIntosh** [comment]
Closed after discussion last night

**Comment by knod** [comment]
Just a reminder that if we ever do both a 'live'/'production' site and a 'dev' site, we'll need the code from this worked out. Also, it would be good to be able to deploy from our own forks, but that's a matter of convenience.

**Pull request closed without merging**